Formula/cgal.rb

```diff
@@ -19,8 +19,7 @@ class Cgal < Formula
 
   def install
     args = std_cmake_args + %W[
-      -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON
-      -DCMAKE_INSTALL_NAME_DIR=#{HOMEBREW_PREFIX}/lib
+      -DCMAKE_CXX_FLAGS='-std=c++11'
```

Comment:
Hopefully, CGAL is header-only, and those `CMAKE_CXX_FLAGS` are actually not used. Because CGAL-5.0 requires C++14, and not C++11.

Reply:
Yes !